README.md

```diff
@@ -33,7 +33,8 @@ Catalog entry:
  :sqs/region "us-east-1"
  :sqs/queue-name queue-name
  :sqs/deserializer-fn :clojure.edn/read-string
- :sqs/attribute-names []
+ :sqs/attribute-names ["All"]
+ :sqs/message-attribute-names ["All"]
```

Comment:
I would like to keep these as [], as we prefer it to be opt in. The amount of message passing / serialization in these systems can add up and thus the best default practice is to only pull it / pass it around if you need it.